nomad/nextcloud.nomad

```diff
@@ -0,0 +1,261 @@
+job "nextcloud" {
+  datacenters = ["dc1"]
+  type = "service"
+  
+  group "nc" {
+    task "nextcloud" {
+      driver = "docker"
+      config {
+        image = "nextcloud:15"
+        volumes = [
+          "/opt/hoover/volumes/nextcloud:/var/www/html",
+          "/opt/hoover/collections/ncsync:/var/www/html/data/ncsync/files",
+        ]
+        port_map {
+          http = 80
+        }
+        labels {
+          liquid_task = "nextcloud"
+        }
+      }
+      env {
+        POSTGRES_DB: nextcloud
+        POSTGRES_USER: postgres
+        POSTGRES_PASSWORD: secret
+        POSTGRES_HOST: nextcloud-pg
+        OC_PASS: secret
+      }
+      resources {
+        network {
+          port "http" {}
+        }
+      }
+      service {
+        name = "nextcloud"
+        port = "http"
+      }
+      }
+
+    task "pg" {
```

Comment:
I generally prefer to have dependencies at the top, but it makes sense to have them lower, out of the way, so it's easy to see the main service. What do you think?

Reply:
Is there a Nomad equivalent to `depends_on:`?  
Actually I can see the benefit from having them at the top too.

Reply:
There is no direct equivalent. But you can link to service dependencies through consul's service registry, like we already have between hoover and snoop, and in other places.